docs/processing.md

```diff
@@ -147,49 +147,54 @@ the design phase. The Python-like language currently assumes statements can only
 
 Remove a forbidden attribute such as `http.request.header.authorization` from spans only
 
-`delete(attributes["http.request.header.authorization"]) from traces`
-`delete(attributes["http.request.header.authorization"])`
+`delete(attributes["http.request.header.authorization"]) from traces`  
+`delete(attributes["http.request.header.authorization"])`  
 
 Remove all attributes except for some
 
-`keep_keys(attributes, "http.method", "http.status_code") from metrics`
-`keep_keys(attributes, "http.method", "http.status_code")`
+`keep_keys(attributes, "http.method", "http.status_code") from metrics`  
+`keep_keys(attributes, "http.method", "http.status_code")`  
 
 Reduce cardinality of an attribute
 
-`replace_wildcards("/user/*/list/*", "/user/{userId}/list/{listId}", attributes["http.target"]) from traces`
-`replace_wildcards("/user/*/list/*", "/user/{userId}/list/{listId}", attributes["http.target"])`
+`replace_match(attributes["http.target"], "/user/*/list/*", "/user/{userId}/list/{listId}") from traces`  
+`replace_match(attributes["http.target"], "/user/*/list/*", "/user/{userId}/list/{listId}")`  
 
 Reduce cardinality of a span name
 
-`replace_wildcards("GET /user/*/list/*", "GET /user/{userId}/list/{listId}", name) from traces`
-`replace_wildcards("GET /user/*/list/*", "GET /user/{userId}/list/{listId}", name)`
+`replace_match(name, "GET /user/*/list/*", "GET /user/{userId}/list/{listId}") from traces`  
+`replace_match(name, "GET /user/*/list/*", "GET /user/{userId}/list/{listId}")`  
+
+Reduce cardinality of any matching attribute
+
+`replace_all_matches(attributes, "/user/*/list/*", "/user/{userId}/list/{listId}") from traces`  
+`replace_all_matches(attributes, "/user/*/list/*", "/user/{userId}/list/{listId}")`  
 
 Decrease the size of the telemetry payload by removing large resource attributes
 
-`delete(resource.attributes["process.command_line"]) from traces`
-`delete(resource.attributes["process.command_line"])`
+`delete(resource.attributes["process.command_line"]) from traces`  
+`delete(resource.attributes["process.command_line"])`  
 
 Filtering out signals such as by removing all metrics with a `http.target` of `/health`
 
-`drop() where attributes["http.target"] = "/health" from metrics`
-`if attributes["http.target"] = "/health": drop()`
+`drop() where attributes["http.target"] = "/health" from metrics`  
+`if attributes["http.target"] = "/health": drop()`  
```

Comment:
How do we say the same thing here that only drop attributes from "metrics" points not spans (second line does not have anything to signal that).

Reply:
I think some of this is holdover from the original issue/pr discussion where it was being debated how the query language would interact with different signals.  The doc currently says "The Python-like language currently assumes statements can only be applied to one signal.", which at the same time implies that the SQL-like language could be applied across multiple.

In reality the collector architecture (and as a result the transform processor) won't let a function work on multiple signals at a time.  I think we could remove all the `from ...` statements (simplifying the comparison examples where possible) and update the doc to state a query can only work on 1 signal at a time.

Reply:
Yeah we can probably remove them now and even the python-like syntax, don't think we have real plans to experiment with it at this point

Reply:
@bogdandrutu opened up #5408 to address this issue